lib/questionnaires/view/src/questionnaire_theme.dart

```diff
@@ -33,6 +33,9 @@ abstract class QuestionnaireTheme {
 
   /// Returns whether user will be offered option for a null radio button value.
   bool showNullCodingOption();
+
+  /// Returns whether each question will be preceded by its own ID.
+  bool showAnswerIndexOption();
```

Comment:
I renamed this to `showQuestionIndexOption`

Honestly, I'm not sure why we're using methods here when we can just as easily set them as a variable that is overridden in the implementation of this abstract class. It just means I have to create two names (`showQuestionIndexOption` + `showQuestionNumbers`) when one could just as easily suffice.

Happy to make that change within this PR, or as a separate...since it's something I had been planning anyways